src/app/util/types_stub.h

```diff
@@ -175,61 +175,22 @@ typedef struct
 /**
  * @brief The broadcast endpoint, as defined in the ZigBee spec.
  */
-#define EMBER_BROADCAST_ENDPOINT (chip::kInvalidEndpointId)
+#define MATTER_BROADCAST_ENDPOINT (chip::kInvalidEndpointId)
 
 /**
  * @brief Useful to reference a single bit of a byte.
  */
-#define EMBER_BIT(nr) (1UL << (nr)) // Unsigned avoids compiler warnings re EMBER_BIT(15)
+#define MATTER_BIT(nr) (1UL << (nr)) // Unsigned avoids compiler warnings re MATTER_BIT(15)
 
 /**
  * @brief Returns the low byte of the 16-bit value \c n as an \c uint8_t.
  */
-#define EMBER_LOW_BYTE(n) ((uint8_t) ((n) &0xFF))
+#define MATTER_LOW_BYTE(n) ((uint8_t) ((n) &0xFF))
```

Comment:
Again, instead of adding this API that should not really be used, please either put these macros in `src/app/util/util.cpp` or better yet use the relevant Encoding::LittleEndian bits there.

Reply:
Didn't find a good API to replace both macros, so instead just hard coded them in the only place they were used

Reply:
you're completely right, somehow I was looking for something else (the >> operator) Will fix